cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -157,6 +159,38 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         // AssessmentQuestionnaire.delete(cloudNative);
     });
 
+    it("Assess and review application associated with unassessed/unreviewed archetypes", function () {
```

Comment:
Can this TC be combined with by adding some steps ?

```it("Assess application and overide assessment for that archetype", function () {```

The steps seems to be covered in lot of other test cases where you assess application and check status .

Reply:
I) Assess application and override assessment for that archetype
2) Assess and review application associated with unassessed/unreviewed archetypes

These two have to be separate test, but I plan to combine another test with # 2 after this PR is merged .